docs/mms_on_fargate.md

```diff
@@ -0,0 +1,256 @@
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/5.+Configure+Routing+(DUmmy).png)
+
+9. nothing need to be done on the last two steps. Finish the creation and ...
+10. now you are ready to remove dummy listener and target group
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/8__Delete_the_dummy_listener.png)
+
+Finally we are done-done-done with the LB, time to move to the service creation
+
+## Create a ECS Service from the ECS Task Definitions
+
+1. Go to Elastic Container Service → Task Definitions and select the task definitions name. Click on actions and select create service.
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/1.+Go+to+task+definitions.png)
+
+2.  There are two important things on the first step (apart from naming):
+
+* platform version, it should be set to 1.1.0 and
+* number of tasks that the service should maintain as healthy all the time, in our case we will use 3
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/number+of+tasks.png)
+
+3. Now it is time to configure the VPC and the security group. One should use EXACTLY same VPC that was used for the LB (and same subnets!)
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.2.1+Use+the+existing+VPC+Edit+sg.png)
+
+4. As for the security group, it should be either the same security group as you had for the LB, or the one that LB have access to:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.2.2+SG+Use+existing.png)
+
+5.  Now we can connect our future service to the existing LB. Select the application load balancer and set the LB name:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.2.3+Add+load+balancing.png)
+
+6. Now we need to specify which port on the LB our service should be listening on:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.2.4+Configure+load+blancer.png)
+
+7. We are not going to use service discovery now, so uncheck it:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.2.5+Next.png)
+
+8. Also for the sake of the simplicity we are not going to use auto-scaling functionality
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.3+Auto+scaling.png)
+
+9. Now we are done-done-done and we can move to the final chapter of our journey:
+
+## Test The Inference
+
+First let's figure out the DNS name of our LB. It should be in EC2 => Load Balancers and click on the LB:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/lb_dns.png)
+
+Now we can run the health checks to verify that it is working:
+```bash
+➜ ~ curl InfraLb-1624382880.us-east-1.elb.amazonaws.com/ping 
+
+http://infralb-1624382880.us-east-1.elb.amazonaws.com/ping
+{
+    "health": "healthy!"
+}
+```
+And now we are finally ready to run our inference! Let's download example iamge:
```

Comment:
you are

run inference

Download an example image:

Reply:
Done